phpstan.neon

```diff
@@ -26,4 +26,4 @@ parameters:
     ignoreErrors:
         # collectors generics
         - '#Parameter (.*?) of method (.*?)Collector::processNode\(\) should be compatible with parameter#'
-        - '#Method (.*?)Collector::processNode\(\) should return#'
+        - '#Method (.*?)Collector::processNode\(\) should return.*but returns array#'
```

Comment:
this ignore rule made phpstan ignore the error we fix with this PR.

I made it a bit more specific so it will allow us catching similar errors in the future

Reply:
in a followup I was able to drop the whole ignore